src/test/java/com/wikia/webdriver/common/core/configuration/Configuration.java

```diff
@@ -141,32 +146,40 @@ public static String getEmulator() {
     return getProp("emulator");
   }
 
-  public static String getPageLoadStrategy(){
+  public static String getPageLoadStrategy() {
     return getProp("unstablePageLoadStrategy");
   }
 
   public static Credentials getCredentials() {
     return new Credentials();
   }
 
-  public static String getEnvType() {
+  public static EnvType getEnvType() {
     if (getEnv().contains("prod")) {
-      return "prod";
+      return EnvType.PROD;
+    } else if (getEnv().contains("staging")) {
+      return EnvType.STAGING;
     } else if (getEnv().contains("verify") || getEnv().contains("preview")
-               || getEnv().contains("sandbox")) {
-      return "staging";
+        || getEnv().contains("sandbox")) {
+      return EnvType.SANDBOX;
     } else if (getEnv().contains("dev")) {
-      return "dev";
+      return EnvType.DEV;
     }
-    return "";
-  }
-
-  public static String getWikiaDomain() {
-    if (StringUtils.isBlank(wikiaDomain)) {
-      wikiaDomain = "dev".equals(getEnvType()) ? ".wikia-dev.com" : ".wikia.com";
+    return EnvType.PROD;
+  }
+
+  public static EnvType getEnvType(String env) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "getEnvType" is 11 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)